src/main/java/Command.java

```diff
@@ -0,0 +1,40 @@
+public class Command {
+    String[] info;
+    boolean add;
+    boolean delete;
+    boolean done;
+    boolean isExit = false;
+
+    public Command(String[] info, boolean add, boolean delete, boolean done){
+        this.info = info;
+        this.add = add;
```

Comment:
I think that it would be great if you followed how you name your boolean values in `Duke.java`. In particular, using `'isAdd`, `isDelete`, etc. would make the purpose of theses variables clearer for me.

Reply:
I've changed it in my recent increments! thanks for the suggestion